### ISSUE TITLE
Blocked by Traffic Welker Farm map field 2

### ISSUE THREAD

**Issue by WolfmoonCT** [Courseplay/courseplay#2790]
I'm guessing this is a map issue over the mod.. but on the new Welker Farm map, field 2 when trying to seed it, it will randomly stop in the middle of the field saying blocked by traffic, even though it's not near a road, and traffic is turned off...  any idea?

**Comment by pops64** [comment]
What version are you running? Are there other things on the field? Does something drive by it randomly?

**Comment by WolfmoonCT** [comment]
Mod is 5.3.0  with map v1.0.0.0.  There is nothing else going by.. the map is HUGE and there are no public roads on the map, so no traffic.   The field is not a rectangle.  I had it set to do 2 headlands and it looked like it was doing it.  then it got to a point in the middle of a field and it said it was blocked by traffic. I took over and pretty much any place on that field it says it's blocked by traffic.  

I tried it on two rectangle ones and it worked fine. Right now it's running on a field that the mission gives 260 min to complete with a combine..  It did one headland, and now it's just doing back and forth passes and so far it's chugging along.

**Comment by pops64** [comment]
Log with debug channel 3 and only that tractor active(Channel 3 is very busy so if more that 1 CP is running with this channel it gets hard to find any problems). Also after you do this, If you know how to enable developer mode post a picture with F5 active where it is stopping, If i were to guess there is something not able to be seen with out that active that the traffic is picking up.

**Comment by WolfmoonCT** [comment]
Honestly, I haven't a clue. I've been playing the game for about two weeks, and only started using mods about a week ago, and I've only tried Courseplay once on a different map..  I have no clue about enabling developer mode.

**Comment by pops64** [comment]
Just logs and a picture of where it stops with debug channel 3 then. Thanks

**Comment by WolfmoonCT** [comment]
Will do. I may not get to it till this weekend, but I will upload it and provide it for you.  Thanks.

**Comment by pops64** [comment]
Is it encountering errors during the missions only or both? Has CP is not supported to be used during field work missions

**Comment by WolfmoonCT** [comment]
So far I have harvested, plowed, fertilized, cultivated, and replanted without the issues showing up again.  I'm going to close this, as maybe it was just some strange glitch cause I started the map with traffic on (since it's the default) and after it gave the first error about being stuck in traffic, I turned it off, but did not restart the map.  Maybe that was still an issue cause I hadn't saved and restarted.  Thanks for your help though.
